test/integration/conftest.py

```diff
@@ -40,13 +40,12 @@ def container_runtime_available():
 # TODO: determine if we want to add docker / podman
 # to zuul instances in order to run these tests
 @pytest.fixture(scope="session", autouse=True)
-def container_runtime_installed(container_runtime_available):
+def container_runtime_installed():
     import subprocess
 
-    if container_runtime_available:
-        for runtime in ('podman', 'docker'):
-            try:
-                subprocess.run([runtime, '-v'])
-                return runtime
-            except FileNotFoundError:
-                pass
+    for runtime in ('podman', 'docker'):
+        try:
+            subprocess.run([runtime, '-v'])
+            return runtime
```

Comment:
The fixture `container_runtime_available` returns True if, and only if, _both_ docker and podman are installed. Well if that's the case, you don't need to run this loop because I can tell you the answer. It's podman.

Assuming that this had a point other than returning either "podman" or None, I made it so that it can return "docker" if that is there but podman is not.